phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -96,6 +100,30 @@ public void showAvailableApps() {
      */
     public void populate(List<CategoryDTO> categories) {
         Platform.runLater(() -> {
+            // set default category icons
+            try{
+                String css = "";
+                for (CategoryDTO category : categories) {
+                    css += "." + category.getName().toLowerCase() + "Button{\n";
```

Comment:
As far as I understand our categories, there can be at most one category with the same class/name or?
If this is the case I think it's better to use a css-id instead of the class.